scala-mode.el

```diff
@@ -176,6 +176,8 @@ When started, runs `scala-mode-hook'.
 (progn
   (add-to-list 'auto-mode-alist
                '("\\.\\(scala\\|sbt\\|worksheet\\.sc\\)\\'" . scala-mode))
+  (add-hook 'scala-mode-hook
+            (lambda () (setq-local adaptive-fill-regexp "[ \t]*\\(//+[ \t]*\\)*")))
```

Comment:
I don't think this is the correct place.

You should set it up where all the other things are being set, i.e. near line 165, for example

Reply:
`setq-local` is a function that creates a local variable and then sets it.

In `scala-mode.el` we first create the local variables starting at line 108, then sets the value starting from line 136.

Thus if you follow the same pattern, you will be good.

Reply:
I did some testing and it confirms your answer. I am about to force push an update here that will make a lot more sense.

Reply:
Ah yes, this implementation makes so much more sense. Glad you educated me. 😁